docs/blog/2026-08-29-migrate-off-react-bulma-components.md

```
@@ -0,0 +1,245 @@
+  sizeDesktop={4}
+  offsetDesktop={2}
+>
+  Side
+</Column>
+```
+
+Three of the four breakpoint objects flattened cleanly. The fourth, `touch`, has no equivalent in bestax, so it was left exactly where it was, with a comment naming the problem and a suggested way out. Not deleted, not approximated into something that looks close. Left alone, and reported.
+
+Every one of those comments also lands in the run's report, grouped by rule and sorted by how often it fired. Here's a slice of a real run against the project's own kitchen-sink fixture app:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)